org/wildfly/mod_cluster/capability.adoc

```diff
@@ -0,0 +1,19 @@
+NAME: org.wildfly.mod_cluster
```

Comment:
Service names vs capabilities, injection of capabilities are the two biggest sore points.